zookeeper-api/src/main/java/org/apache/helix/zookeeper/constant/ZkSystemPropertyKeys.java

```diff
@@ -50,4 +50,14 @@
    */
   public static final String ZK_SERIALIZER_ZNRECORD_WRITE_SIZE_LIMIT_BYTES =
       "zk.serializer.znrecord.write.size.limit.bytes";
+
+  /**
+   * This property determines the behavior of ZkClient issuing an sync() to server upon new session
+   * established.
+   *
+   * <p>
+   *   The default value is "true" (issuing sync)
+   */
+  public static final String ZK_AUTOSYNC_ENABLED =
+      "zk.autosync.enabled";
```

Comment:
should we name it "zk.zkclient.autosync.enabled"?  zk is higher namespace, it includes configs not only applies to zkClient, it could be all zk-related configs in Helix.

Reply:
good point. changed.